changelog.d/214.misc

```diff
@@ -1 +1 @@
-Upgrade development dependencies fixes #199. Contributed by Omar Mohamed
+Upgrade development dependencies fixes #199. Contributed by Omar Mohamed.
```

Comment:
Oops! I noticed one final thing right before I hit merge 😆

Could you please rename this file to `215.misc` instead of `214.misc`, and maybe change the description to more closely match (e.g., "Add note about pip timeouts. Contributed by Omar Mohamed.")

Reply:
On it :smiley:

Reply:
215.doc here would be fine :)

Reply:
I have made changes on the `215.doc` changelog file with a more descriptive message on `pip timeout` :smiley: